pyensembl/database.py

```diff
@@ -81,25 +82,42 @@ def _create_database(self):
             indices=indices)
         return db
 
-    def _connect_or_create_database(self):
+    def connection_if_exists(self):
```

Comment:
I made this one "public" because it's used by `EnsemblRelease`; that makes sense to make `_connect_or_create_database` "public".